packages/tools/pie-monorepo-utils/utils/find-monorepo-root.js

```diff
@@ -0,0 +1,16 @@
+const fs = require('fs');
+const path = require('path');
+
+function findMonorepoRoot (startPath = process.cwd(), expectedFile = 'yarn.lock') {
```

Comment:
We have a couple functions like this across the repo. If this PR gets merged, I'll create a follow-up JIRA to ensure other packages utilise this version instead.

Reply:
@siggerzz Can we make sure to test this on Windows to make sure the file path logic holds up? I'm happy to help with this